packages/backend/src/peripherals/coingecko/CoingeckoQueryService.ts

```diff
@@ -89,6 +89,7 @@ export function pickPrices(
   prices: { price: number; date: Date }[],
   timestamps: UnixTime[]
 ): PriceHistoryPoint[] {
+  if (prices.length === 0) return []
```

Comment:
This seems like a temporary workaround. It's fine to keep it, but I'd add a comment: // TODO: Handle this case properly